federation/normalize.go

```diff
@@ -0,0 +1,278 @@
+package federation
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+
+	"github.com/samsarahq/thunder/graphql"
+)
+
+func collectTypes(typ graphql.Type, types map[graphql.Type]string) error {
```

Comment:
This function is also in `schema/normalize.go`, can we remove it from there?

Reply:
done